instrumentation/servlet/servlet-javax-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/servlet/javax/response/ResponseTracer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.servlet.javax.response;
+
+import io.opentelemetry.context.Context;
+import io.opentelemetry.instrumentation.api.tracer.BaseTracer;
+import java.lang.reflect.Method;
+
+public class ResponseTracer extends BaseTracer {
```

Comment:
This is identical to `RequestDispatcherTracer`. Should they be de-duplicated?

Reply:
Possibly, but not sure what a good name would be in that case since it would just be a generic tracer with no special behavior other than instrumentation name.